rpa-manager/modules/ROOT/pages/processautomation-manage-botworkflowlibrary.adoc

```diff
@@ -0,0 +1,112 @@
+= Managing the Bot Workflow Library
+
+The *Bot Workflow Library* view shows all bot workflows in the RPA Builder Activity Library.
+
+You can do the following to view and manage the workflows:
+
+* Create, edit, and delete tags for the library.
+* View an outline of a workflow.
+* Edit the description and the tags of a workflow.
+
+== Before You Begin
+
+* Ask an organization administrator in Access Management to assign you the required permissions:
++
+[cols="1,1,1"]
+|===
+|*Action* |*RPA Permission* | *Deprecated RPA Permission*
+
+|Open the *Process Automation* module.
+|RPA Administrator
+|Process Automation Open
+
+|Open the Bot Workflow Library.
+|RPA Automations Contributor, RPA Automations Manager
+|Activity Library Open
+
+a|
+* Create tags for the library. 
+* Add, edit, and remove tags from bot workflows.
+* Add and edit bot workflow descriptions.
+|RPA Automations Manager
+|Activity Library Administration
+
+|===
+
+== Viewing Bot Workflows
+
+Get an overview of all bot workflows.
+
+=== View the List of All Bot Workflows
+
+Open the *Bot Workflow Library* view in the *Process Automation* module to view the list of all bot workflows and see their purpose.
+
+=== View an Outline of a Bot Workflow
+
+View the outline of a bot workflow to better understand how it works:
+
+. Open the *Bot Workflow Library* view in the *Process Automation* module.
+. Click *Details* image:show-icon.png[eye symbol,1.5%,1.5%] in the panel of the workflow to view.
```

Comment:
For the inline images, I believe there should be parenthesis around them. From the screenshot guide:

Click the Run Now icon (image:icon.png[]).

https://confluence.internal.salesforce.com/display/MTDT/Create+Desktop+Screenshots+for+MuleSoft+Docs